docker/pdo_ccf.dockerfile

```diff
@@ -44,7 +46,9 @@ WORKDIR /project/pdo/tools
 COPY --chown=${UNAME}:${UNAME} tools/*.sh ./
 
 # build it!!!
-RUN /project/pdo/tools/build_ccf.sh
+ARG UID=1000
+ARG GID=${UID}
+RUN --mount=type=cache,uid=${UID},gid=${GID},target=/project/pdo/.cache/pip /project/pdo/tools/build_ccf.sh
 
```

Comment:
only suggestion is that 1) add comments explaining what this does and why it works and 2) split the line into the mount parameters and the actual command that is run.